docs/Halogen-HTML.md

```diff
@@ -1742,6 +1777,84 @@ wbr_ :: forall p i node. (HTMLRepr node) => [node p i] -> node p i
 This module enumerates some common HTML attributes, and provides additional
 helper functions for working with CSS classes.
 
+#### `ExistsR`
+
+``` purescript
+data ExistsR (f :: # * -> *)
```

Comment:
Possibly, it's a real shame we don't have `PolyKinds`. `Exists` would make such a nice application of it. I thought it best not to move it there, since then we would need to support `Exists` for some (suitably small but) arbitrary set of kinds. I'd rather wait for `PolyKinds` honestly.